cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/survey/grade-level-3-5.html

```diff
@@ -26,7 +26,8 @@
                 <span class="a-link_text">Switch grades</span>
             </a>
             <div class="block block__flush-bottom">
-                <span class="h3"><img src="{{ static('apps/teachers-digital-platform/img/dollar.svg') }}" alt="dollar icon"></span><span class="h3" style="font-weight: 500;"> {{ survey.meta.name }}</span>
+                <span class="h3">{% include 'icons/money-round.svg' %}</span>
+                <span class="h3" style="font-weight: 500;"> {{ survey.meta.name }}</span>
```

Comment:
We should not be introducing inline styling like this. What is this PR about? There is no info filled in?

Reply:
We're using PRs for review within our tiny team while building the TDP survey system. 
@karenkitchens Let's standardize on a title like `[WIP] TDP survey progress` for these PRs and use an empty description. The change descriptions should be in the commit messages.

@anselmbradford Is this a reasonable compromise?

Reply:
@anselmbradford https://github.com/cfpb/consumerfinance.gov/pull/6511 is the main PR being assembled for the survey system. If draft PRs still get dev environments then I can make that a draft.

Reply:
Ohh I see, I overlooked that it is not going into `main`.